packages/fxa-graphql-api/src/config.ts

```diff
@@ -6,116 +6,170 @@ import convict from 'convict';
 import fs from 'fs';
 import path from 'path';
 
-const conf = convict({
-  authHeader: {
-    default: 'authorization',
-    doc: 'Authentication header that should be logged for the user',
-    env: 'AUTH_HEADER',
-    format: String,
-  },
-  database: {
+export interface MySQLConfig {
+  database: string;
+  host: string;
+  password: string;
+  port: number;
+  user: string;
+}
+
+export interface RedisConfig {
+  host: string;
+  keyPrefix: string;
+  port: number;
+}
+
+function makeMySQLConfig(envPrefix: string, defaultDatabase: string) {
```

Comment:
Could name it `database`, a default value wouldn't work since its expected you could put multiple ones of these in a single project (like here), in which case they must be unique for each connection.